src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -329,32 +329,35 @@ public Command tuneSetpoints(
 
   @Override
   public void periodic() {
+    if (!Robot.isCompetition) {
+      Logger.recordOutput(
+          "Elevator/Motor1/Position (Degrees)",
+          elevatorMotor1.getPosition().getValue().in(Degrees));
+
+      Logger.recordOutput(
+          "Elevator/Motor2/Velocity", elevatorMotor2.getVelocity().getValue().in(DegreesPerSecond));
+
+      Logger.recordOutput("Elevator/Elv/Height (Inches)", getElevatorHeight().in(Inches));
+      Logger.recordOutput("Elevator/limit switch state", elvLimitSwitch.get());
```

Comment:
We should keep Height and limit switch logged during Comp
All of Motor2 logs can probably be moved to the if block